src/iris/api.py

```diff
@@ -4732,6 +5194,10 @@ def construct_falcon_api(debug, healthcheck_path, allowed_origins, iris_sender_a
     api.add_route('/v0/response/slack', ResponseSlack(iris_sender_app))
     api.add_route('/v0/twilio/deliveryupdate', TwilioDeliveryUpdate())
 
+    api.add_route('/v0/categories', NotificationCategories())
+    api.add_route('/v0/categories/{category_id}', NotificationCategory())
+    api.add_route('/v0/users/{username}/categories/{application}', CategoryOverrides())
```

Comment:
It might be useful to add a /v0/users/{username}/categories endpoint to list all the overrides a user has defined together, or at least all the applications for which a user has defined overrides for. This would be useful for populating the user settings front end without having to make separate calls for every  application.